dts/riscv/ite/it8801-common-cfg.dtsi

```diff
@@ -6,149 +6,145 @@
 
 #include <ite/it8801-mfd-map.dtsi>
 
-&it8801_i2c {
-	it8801_mfd: it8801@38 {
-		compatible = "ite,it8801-mfd";
-		reg = <0x38>;
-		#address-cells = <1>;
-		#size-cells = <1>;
+&it8801_mfd {
```

Comment:
Please update the help in ite,it8801-mfd.yaml to reflect the recommend usage.

Reply:
Good point, done.